src/main/java/com/LuuQu/medicalclinicproxy/client/AppointmentClient.java

```diff
@@ -0,0 +1,26 @@
+package com.LuuQu.medicalclinicproxy.client;
+
+import com.LuuQu.medicalclinicproxy.configure.AppointmentFallbackFactory;
+import com.LuuQu.medicalclinicproxy.configure.FeignConfig;
+import com.LuuQu.medicalclinicproxy.model.AppointmentDto;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@FeignClient(name = "appointmentClient", url = "http://medicalclinic:8080/appointments",
```

Comment:
Okej takie rozwiązanie zadziała w przypadku konteneru ale jak odpalimy lokalnie aplikację już nie -> moglibyśmy zdefiniować tutaj tzw urla parametryzowanego -> w application properties zdefiniowalibysmy base url dla tego klienta bądź możemy wstrzyknąć tutaj jakiegoś urla tak aby był on konfigurowalny.
Dzięki takiemu zabiegowi następnie w docker compose byśmy sobie odpalając kontener z tą aplikacją wstrzykiwali nowa wartosc tego propertisa 
Czyli lokalnie w application.properties mielibysmy urla jako: 
http://localhost:8080/appointments

Natomiast odpalajac docker compose nadpisywalibysmy to:
http://medicalclinic:8080/appointments przekazujac w sekcji environments w docker compose